static/app/components/events/packageData.tsx

```diff
@@ -39,13 +49,64 @@ export function EventPackageData({event}: Props) {
     return null;
   }
 
+  const componentItems = packages.map((item, i) => (
+    <KeyValueData.Content
+      key={`content-card-${item.key}-${i}`}
+      item={item}
+      meta={item.meta}
+    />
+  ));
+
+  const columns: React.ReactNode[] = [];
+  const columnSize = Math.ceil(componentItems.length / columnCount);
+  for (let i = 0; i < componentItems.length; i += columnSize) {
+    columns.push(
+      <Column key={`highlight-column-${i}`}>
+        {componentItems.slice(i, i + columnSize)}
+      </Column>
+    );
+  }
+
   return (
-    <InterimSection title={title} type={SectionKey.PACKAGES}>
-      <ClippedBox>
-        <ErrorBoundary mini>
-          <KeyValueList data={packages} longKeys={longKeys} />
-        </ErrorBoundary>
-      </ClippedBox>
+    <InterimSection
+      title={title}
+      type={SectionKey.PACKAGES}
+      ref={containerRef}
+      initialCollapse
+    >
+      {hasStreamlinedUI ? (
+        <ColumnsContainer columnCount={columnCount}>{columns}</ColumnsContainer>
+      ) : (
+        <ClippedBox>
+          <ErrorBoundary mini>
+            <KeyValueList data={packages} longKeys={longKeys} />
+          </ErrorBoundary>
+        </ClippedBox>
+      )}
     </InterimSection>
   );
 }
+
+export const ColumnsContainer = styled('div')<{columnCount: number}>`
+  display: grid;
+  align-items: start;
+  grid-template-columns: repeat(${p => p.columnCount}, 1fr);
+`;
+
+export const Column = styled('div')`
+  display: grid;
+  grid-template-columns: fit-content(50%) 1fr;
```

Comment:
might be able to get away with more than 50% here since package names tend to be longer than the version